packages/node/test/module.test.ts

```diff
@@ -0,0 +1,15 @@
+import { getModule } from '../src/module';
+
+describe('getModule', () => {
+  test('Windows', async () => {
+    (require.main as any) = { filename: 'C:\\Users\\Tim\\app.js' };
```

Comment:
can we mock this return value with jest, and `mockRestore` afterwards?

Reply:
`jest.spyOn` complains that `require` and `require.main` are "not declared configurable". I guess this means the properties are marked as `configurable: false` and there is no way to modify them other than by overwriting them?